src/Storage/Storage.Management.Test/ScenarioTests/StorageDataPlaneTests.ps1

```diff
@@ -821,7 +821,9 @@ function Test-DatalakeGen2
 		$dir1 = New-AzDataLakeGen2Item -Context $storageContext -FileSystem $filesystemName -Path $directoryPath1 -Directory -Permission rwxrwxrwx -Umask ---rwx---  -Property @{"ContentEncoding" = "UDF8"; "CacheControl" = "READ"} -Metadata  @{"tag1" = "value1"; "tag2" = "value2" }
 		Assert-AreEqual $dir1.Path $directoryPath1
         Assert-AreEqual $dir1.Permissions.ToSymbolicPermissions() "rwx---rwx"
-		$dir2 = New-AzDataLakeGen2Item -Context $storageContext -FileSystem $filesystemName -Path $directoryPath2 -Directory
+		$dir2 = New-AzDataLakeGen2Item -Context $storageContext -FileSystem $filesystemName -Path $directoryPath2 -Directory -Permission r---wx-wT -Umask --x-wx--x
+        Assert-AreEqual $dir2.Path $directoryPath2
+        Assert-AreEqual $dir2.Permissions.ToSymbolicPermissions() "r------wt"
```

Comment:
Correct indents please

Reply:
Sorry I was having some trouble adjusting the indentation here, as the indentation in VS looks fine... 
![image](https://user-images.githubusercontent.com/100746763/180391381-d993b263-c311-4b41-ba94-a8702612b075.png)
Do you happen to know how to deal with this?

Reply:
Also just checked this test file on main - it seems like it's misaligned everywhere while it all looks fine locally on my side :( 
https://github.com/Azure/azure-powershell/blob/main/src/Storage/Storage.Management.Test/ScenarioTests/StorageDataPlaneTests.ps1#L821

Reply:
Yes, please use VS code and use it to convert all tabs to spaces. Then you can find the difference

Reply:
Done. Thanks a lot for the help :)